app-vcs/brz/autobuild/build

```diff
@@ -0,0 +1,4 @@
+python3 setup.py build
+python3 setup.py install --prefix=/usr --root="$PKGDIR" --optimize=1
+install -D -m644 contrib/bash/brz \
+    $PKGDIR/usr/share/bash-completion/completions/brz
```

Comment:
A suggestion would be to create a symlink to `bzr`? Since it's really considered a drop-in replacement, can you confirm?

This way, we wouldn't have to modify KDevelop.